src/gtest/test_forkmanager.cpp

```diff
@@ -292,7 +421,7 @@ TEST(ForkManager, SidechainVersionForkMainnet) {
     SelectParams(CBaseChainParams::MAIN);
 
     // TODO: set proper fork height value.
```

Comment:
This comment can be removed.
```suggestion
```

Reply:
Addressed in https://github.com/HorizenOfficial/zen/commit/6f1ffaeccf187e2903bfa608c54c5138fe7d834c